src/store/index.js

```diff
@@ -1,28 +1,69 @@
 import { create } from "zustand";
 import { devtools } from "zustand/middleware";
 
-const urlStore = (set) => ({
-  videoUrls: { mainVideoUrl: "", firstSubVideoUrl: "", lastSubVideoUrl: "" },
-  audioUrls: { mainAudioUrl: "", firstSubAudioUrl: "", lastSubAudioUrl: "" },
+const awsUrlStore = (set) => ({
+  videoUrls: {
+    mainVideoUrl: "",
+    subOneVideoUrl: "",
+    subTwoVideoUrl: "",
+  },
+  audioUrls: {
+    mainAudioUrl: "",
+    subOneAudioUrl: "",
+    subTwoAudioUrl: "",
+  },
+  setVideoUrls: (mainVideoUrl, subOneVideoUrl, subTwoVideoUrl) =>
+    set({
+      videoUrls: {
+        mainVideoUrl,
+        subOneVideoUrl,
+        subTwoVideoUrl,
+      },
+    }),
+  setAudioUrls: (mainAudioUrl, subOneAudioUrl, subTwoAudioUrl) =>
+    set({
+      audioUrls: {
+        mainAudioUrl,
+        subOneAudioUrl,
+        subTwoAudioUrl,
+      },
+    }),
+});
```

Comment:
현재 하나의 스토어에서 오디오와 비디오의 모든 정보를 관리하고 있는데 오디오와 비디오의 스토어를 분리하는 것도 좋을 것 같다는 의견입니다! 보다 관리가 용이할 것 같습니다 :)

```suggestion

const awsVideoStore = (set) => ({
  videoUrls: {
    mainVideoUrl: "",
    subOneVideoUrl: "",
    subTwoVideoUrl: "",
  },
  setVideoUrls: (mainVideoUrl, subOneVideoUrl, subTwoVideoUrl) =>
    set({
      videoUrls: {
        mainVideoUrl,
        subOneVideoUrl,
        subTwoVideoUrl,
      },
    }),
});

const awsAudioStore = (set) => ({
  audioUrls: {
    mainAudioUrl: "",
    subOneAudioUrl: "",
    subTwoAudioUrl: "",
  },
  setAudioUrls: (mainAudioUrl, subOneAudioUrl, subTwoAudioUrl) =>
    set({
      audioUrls: {
        mainAudioUrl,
        subOneAudioUrl,
        subTwoAudioUrl,
      },
    }),
});
```

Reply:
스토어가 너무 뚱뚱해지면 관리도 어려워지고 관심사가 불명확해지는데 분리하니 그런 불편함에 해소될 수 있겠네요!